.github/workflows/sgx.yml

```diff
@@ -0,0 +1,56 @@
+name: sgx
+
+on:
+  push:
+    branches: [ 'sgx**' ]
+  pull_request:
+    branches: [ 'sgx**' ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-18.04
+    container: 
+      image: baiduxlab/sgx-rust:1804-1.1.1
+
+    steps:
+    - name: Upgrade git to >=2.18 for checkout@v2
+      run: |
+        add-apt-repository ppa:git-core/ppa -y &&\
+        apt update &&\
+        apt install -y git
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'true'
+    - name: Set up Github Actions' $HOME
+      run: |
+        cp /root/.bashrc $HOME/.bashrc &&
+        ln -sf /root/.rustup ~/.rustup &&
+        ln -sf /root/.cargo ~/.cargo
+    - name: Rust toolchains
+      run: |
+        . ~/.cargo/env && 
+        rustc --version &&
+        rustup --version &&
+        cargo --version
+      shell: bash
+    - name: Cache cargo registry
+      uses: actions/cache@v1
+      with:
+        path: ~/.cargo/registry
+        key: ${{ runner.os }}-cargo-registry-${{ hashFiles('**/Cargo.toml') }}
+        restore-keys: ${{ runner.os }}-cargo-registry- 
+    - name: Cache cargo index
+      uses: actions/cache@v1
+      with:
+        path: ~/.cargo/git
+        key: ${{ runner.os }}-cargo-index-${{ hashFiles('**/Cargo.toml') }}
+        restore-keys: ${{ runner.os }}-cargo-index- 
+    - name: Run tests
+      run: |
+        . ~/.cargo/env && 
+        . /opt/sgxsdk/environment && 
```

Comment:
I believe this is not needed if using this `teaclave/teaclave-build-ubuntu-1804-sgx-2.9:0.1.0` image.

Reply:
I think not. This line addressed the hardcoded HOME (/home/github) directory set by Github Actions. @mssun